plugins/flutter_plugins/url_launcher_fde/pubspec.yaml

```diff
@@ -7,6 +7,8 @@ homepage: https://github.com/google/flutter-desktop-embedding/tree/master/plugin
 flutter:
   plugin:
     platforms:
+      ios:
+        pluginClass: UrlLauncherFakeFdePlugin
```

Comment:
Was this necessary to fix an error? I would not expect this to be needed (or desirable).

Reply:
Interesting, tbh - I'm not sure, I thought it was.. maybe in an older version? anyway, I removed that and tested it on flutters stable and master channel.. and it also makes the actual classes redundant. removed them..